en/docs/learn/references/natural-functions.md

```diff
@@ -116,12 +116,12 @@ Follow the steps below to implement the integration.
         "content": "For those who want a 360-degree approach to self-care, with advice for betterment in the workplace, home, gym, and on the go, look no further. The Healthy Maven offers recipes for every type of meal under the sun (salads, sides, soups, and more), DIY tips (you’ll learn how to make your own yoga mat spray), and quick workouts. If you like where all this is going, there’s a supplementary podcast run by blogger Davida with guest wellness experts."
     }
     ```
-5. The response will be a review of the blog content with the category and rating.
+5. The response will be a review of the blog content with the category and rating (The rating value may not always be identical across different executions).
```

Comment:
I think the fact that the rating can change is obvious. Technically category can too. So if we are saying something we should say it all.

Reply:
Will remove that